### DISCUSSION TITLE
milestone for release tracking

### DISCUSSION THREAD

**Discussion by srliao** [genshinsim/gcsim#1521] [General]
@unleashurgeek @k0l11 would it be a good idea to use milestones to track minor and major releases? for example, enka automation could be the next minor release?

Might help avoid missing issues as well?

**Comment by k0l11** [comment]
Maybe, if it was kept up-to-date. I don't really see a major issue with the way we do releases right now.

**Reply by srliao** [reply]
no major issues. just easier to keep track if not paying attention to discord
